tango_ros_common/tango_ros_native/test/test_tango_ros_msg.cpp

```diff
@@ -85,44 +88,25 @@ class TangoRosTest : public ::testing::Test {
     point_cloud_message_count_ ++;
   }
 
-  void FisheyeImageCallback(const boost::shared_ptr<const sensor_msgs::CompressedImage> msg) {
+  void FisheyeImageCallback(const sensor_msgs::ImageConstPtr& image) {
     fisheye_image_received_ = true;
     fisheye_image_message_count_++;
   }
 
-  void ColorImageCallback(const boost::shared_ptr<const sensor_msgs::CompressedImage> msg) {
+  void ColorImageCallback(const sensor_msgs::ImageConstPtr& image) {
     color_image_received_ = true;
     color_image_message_count_++;
   }
 };
 
 TEST_F(TangoRosTest, TestMessagesArePublished) {
-  tf::StampedTransform transform;
-  EXPECT_THROW(tf_listener_.lookupTransform("/start_of_service", "/device",
-                                            ros::Time(0), transform), tf::TransformException);
-  EXPECT_THROW(tf_listener_.lookupTransform("/device", "/camera_depth",
-                                              ros::Time(0), transform), tf::TransformException);
-  EXPECT_THROW(tf_listener_.lookupTransform("/device", "/camera_fisheye",
-                                              ros::Time(0), transform), tf::TransformException);
-  EXPECT_THROW(tf_listener_.lookupTransform("/device", "/camera_color",
-                                              ros::Time(0), transform), tf::TransformException);
-  EXPECT_FALSE(tf_message_received_);
-  EXPECT_FALSE(point_cloud_received_);
-  EXPECT_FALSE(fisheye_image_received_);
-  EXPECT_FALSE(color_image_received_);
-
   // Sleep some time to be sure that data is published.
   sleep(SLEEP_TIME_UNTIL_FIRST_MESSAGE);
   ros::spinOnce();
 
+  tf::StampedTransform transform;
   EXPECT_NO_THROW(tf_listener_.lookupTransform("/start_of_service", "/device",
                                            ros::Time(0), transform));
-  EXPECT_NO_THROW(tf_listener_.lookupTransform("/device", "/camera_depth",
-                                             ros::Time(0), transform));
-  EXPECT_NO_THROW(tf_listener_.lookupTransform("/device", "/camera_fisheye",
-                                             ros::Time(0), transform));
-  EXPECT_NO_THROW(tf_listener_.lookupTransform("/device", "/camera_color",
-                                             ros::Time(0), transform));
```

Comment:
Why are these tests removed? Maybe also useful to add a test for /area_learning -> /device?